### PR TITLE
feat: welcome toast added to lending home page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
kv-toast component added to lending home page to show a welcome message once per session

### CODE REVIEW

File: src/pages/LoanFinding/LoanFinding.vue [hunk 1]
```diff
@@ -115,11 +132,24 @@ export default {
 		},
 		trackCategory({ success }, category) {
 			if (success) this.$kvTrackEvent('loan-card', 'add-to-basket', `${category}-lending-home`);
+		},
+		closeToast() {
+			this.$refs.welcomeToastMessage.close();
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:137]: Ticket calls out tracking clicks, so make sure to add the tracking events listed in the ticket.

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:137]: right, ty

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:137]: Done

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -115,11 +132,27 @@ export default {
...
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:137]: It looks like there's an issue with the toast refs in this component. There's a max stack depth when this method is called. I logged `this.$refs`, and there are a ton of refs!

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue]: Nice catch! Its fixed.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**